static-lang-word-lists/README.md

```diff
@@ -0,0 +1,26 @@
+# static-lang-word-lists
```

Comment:
Could the readme have an example snippet of Rust code showing how to use one of the word lists? 

Also, does the library allow listing the available word lists, or must the user know which one to use, and know its name? (maybe it's fine for now, whichever way it is, as long as it supports your current use-case)